libraries/eventformatter/impl/src/test/kotlin/io/element/android/libraries/eventformatter/impl/DefaultRoomLastMessageFormatterTest.kt

```diff
@@ -228,7 +239,12 @@ class DefaultRoomLastMessageFormatterTest {
             }
             val shouldCreateAnnotatedString = when (type) {
                 is VideoMessageType -> true
-                is AudioMessageType -> true
+                is AudioMessageType -> {
+                    when (type.isVoiceMessage) {
+                        true -> true
+                        false -> true
```

Comment:
Always `true` then?

Reply:
In the future we'll have to split `AudioMessageType` in two (`AudioMessageType` and `VoiceMessageType`).
When we do this having such structure in place might help a bit.

What do you think?
Should I actually split the message type right now?

Reply:
No worries, if there are plan to change this later :)